### ISSUE TITLE
Extremly weird sourcemaps

### ISSUE THREAD

**Issue by felixfbecker** [TypeStrong/tsify#106]
Project structure:
![image](https://cloud.githubusercontent.com/assets/10532611/12079310/32674030-b234-11e5-8332-177f20e93db6.png)

Sourcemap structure:

![image](https://cloud.githubusercontent.com/assets/10532611/12079304/1790c9a2-b234-11e5-960d-4f7a44f61909.png)

The backslashes is not a problem, I know how to fix them, I just omitted that for pin pointing the issue. For some reason eplan and vplan paths are completely broken.

Heres my gulp code:
```js
browserify('bootstrap.ts', {debug: true, basedir: 'src'})
  .plugin(tsify)
  .transform(babelify, {presets: ["es2015"], extensions: [".ts", ".js"]})
  .bundle()
  .pipe(source('bundle.js'))
  .pipe(buffer())
  ...
```

**Comment by felixfbecker** [comment]
I think it is related to the `basedir` setting.

**Comment by felixfbecker** [comment]
I'm using mold.transformSourcesRelativeTo for now instead of basedir

**Comment by smrq** [comment]
Fixed in 0.14.4
